wremnants/syst_tools.py

```diff
@@ -132,12 +132,14 @@ def projAx(hname):
                     do_min=True)},
        "resumNPUp" : {
             "action" : lambda h: hh.syst_min_or_max_env_hist(h, projAx(hist_name), "vars", 
-                 ["c_nu-0.1-omega_nu0.5", "omega_nu0.5", "Lambda2-0.25", "Lambda20.25", "Lambda4.01", 
+                 #["c_nu-0.1-omega_nu0.5", "omega_nu0.5", "Lambda2-0.25", "Lambda20.25", "Lambda4.01", 
```

Comment:
This is not related to the change in plotting styles, is it intended and does it leave the actual results unchanged?

Reply:
It's intended, it doesn't impact the results just plotting. The variations are way too large and they look problematic in the plots, so I'm just excluding them...